example/lib/src/icon_items.dart

```diff
@@ -20,43 +20,54 @@ final List<IconItem> iconItems = [
 final List<IconItem> animatedIconItems = [
   IconItem(
     name: 'Ok',
-    usage: 'YaruAnimatedOkIcon()',
-    iconBuilder: (context, iconSize) => YaruAnimatedOkIcon(
+    usage: _wrapWithAnimatedIconUsage('YaruAnimatedOkIcon()'),
+    iconBuilder: (context, iconSize) => YaruAnimatedIcon(
+      const YaruAnimatedOkIcon(),
+      mode: YaruAnimationMode.once,
       size: iconSize,
     ),
   ),
   IconItem(
     name: 'Ok filled',
-    usage: 'YaruAnimatedOkIcon(filled: true)',
-    iconBuilder: (context, iconSize) => YaruAnimatedOkIcon(
+    usage: _wrapWithAnimatedIconUsage('YaruAnimatedOkIcon(filled: true)'),
+    iconBuilder: (context, iconSize) => YaruAnimatedIcon(
+      const YaruAnimatedOkIcon(filled: true),
+      mode: YaruAnimationMode.once,
       size: iconSize,
-      filled: true,
     ),
   ),
   IconItem(
     name: 'No network',
-    usage: 'YaruAnimatedNoNetworkIcon()',
-    iconBuilder: (context, iconSize) => YaruAnimatedNoNetworkIcon(
+    usage: _wrapWithAnimatedIconUsage('YaruAnimatedNoNetworkIcon()'),
+    iconBuilder: (context, iconSize) => YaruAnimatedIcon(
+      const YaruAnimatedNoNetworkIcon(),
       size: iconSize,
     ),
   ),
   IconItem(
     name: 'Compass',
-    usage: 'YaruAnimatedCompassIcon()',
-    iconBuilder: (context, iconSize) => YaruAnimatedCompassIcon(
+    usage: _wrapWithAnimatedIconUsage('YaruAnimatedCompassIcon()'),
+    iconBuilder: (context, iconSize) => YaruAnimatedIcon(
+      const YaruAnimatedCompassIcon(),
+      mode: YaruAnimationMode.once,
       size: iconSize,
     ),
   ),
   IconItem(
     name: 'Compass filled',
-    usage: 'YaruAnimatedCompassIcon(filled: true)',
-    iconBuilder: (context, iconSize) => YaruAnimatedCompassIcon(
+    usage: _wrapWithAnimatedIconUsage('YaruAnimatedCompassIcon(filled: true)'),
+    iconBuilder: (context, iconSize) => YaruAnimatedIcon(
+      const YaruAnimatedCompassIcon(filled: true),
+      mode: YaruAnimationMode.once,
       size: iconSize,
-      filled: true,
     ),
   ),
 ];
 
+String _wrapWithAnimatedIconUsage(String source) {
+  return 'YaruAnimatedIcon(const $source)';
+}
+
```

Comment:
Sorry, the idea is good but I don't like how it renders:

![Capture d’écran du 2023-03-28 20-30-48](https://user-images.githubusercontent.com/36476595/228334509-65f66211-bb4d-45c1-91b4-009efd9e21d4.png)

And it is also not consistent with the static icons usage label:

![Capture d’écran du 2023-03-28 20-31-07](https://user-images.githubusercontent.com/36476595/228334501-84eab28f-09e5-44ee-8387-bd264747b6b8.png)

As you see it is indicated `YaruIcons.compass` and not `Icon(YaruIcons.compass)`.
I prefer the simpler current labelling 🙂 

Except this last point, it is LGTM :+1:

Reply:
Sounds fair! Will fix this and commit, then the pr should be good to go :)